docs/bundle-diffs.md

```diff
@@ -0,0 +1,267 @@
+## Generating Diffs for Modified GitRepos
+
+
+Continuous Delivery in Rancher is powered by fleet. When a user adds a GitRepo CR, then Continuous Delivery creates the associated fleet bundles.
+
+You can access these bundles by navigating to the Cluster Explorer (Dashboard UI), and selecting the `Bundles` section.
+
+The bundled charts may have some objects that are amended at runtime, for example in ValidatingWebhookConfiguration the `caBundle` is empty and the CA cert is injected by the cluster.
+
+This leads the status of the bundle and associated GitRepo to be reported as "Modified"
+
+![](./assets/ModifiedGitRepo.png)
+
+Associated Bundle
+![](./assets/ModifiedBundle.png)
+
+Fleet bundles support the ability to specify a custom [jsonPointer patch](http://jsonpatch.com/).
+
+With the patch, users can instruct fleet to ignore the modification on the objects.
```

Comment:
nit: This could be re-worded a little, but the below suggestion might not be the right one.
```
With the patch, users can instruct fleet to ignore object modifications.
```